### DISCUSSION TITLE
Hydrogen as a VSTi trigger within another Host?

### DISCUSSION THREAD

**Discussion by Bjorn218** [hydrogen-music/hydrogen#1306] [Hydrogen on Linux]
Since I cannot use Hydrogen to import any MIDI tracks I have written over the past 20 or so years, Can I use it as a VSTi and load the midi track into one track while having the data send to the Hydrogen track to use Hydrogen as a Triggered sequencer?

MIDI settings of the tracks hydrogen or MIDI file would not matter as they would all be set under how i wan in the Host app.

**Comment by theGreatWhiteShark** [comment]
> Since I cannot use Hydrogen to import any MIDI tracks I have written over the past 20 or so years, Can I use it as a VSTi and load the midi track into one track while having the data send to the Hydrogen track to use Hydrogen as a Triggered sequencer?

Sure. Problem with importing MIDI is that there is no standardized way for representing drumkits more complex then in the general MIDI standard. The way to go is exactly as you described: import the MIDI track in another software, send it to Hydrogen, and configure the MIDI input channels of each instrument in the Instrument Editor to listen to the correct channel.
